.config/docker-compose-base.yaml

```diff
@@ -7,7 +7,7 @@ services:
       context: .
       args:
         grafana_image: ${GRAFANA_IMAGE:-grafana-enterprise}
-        grafana_version: ${GRAFANA_VERSION:-12.0.2}
+        grafana_version: ${GRAFANA_VERSION:-12.1.0}
```

Comment:
cp-update.yml could be used to update the version of grafana, so we'd get this massive upgrade PR once a month, instead of pulling latest and having a random PR fail, wdyt?

Reply:
A lot of devs extend the docker-compose-base.yaml in the docker files https://github.com/grafana/grafana-treemap-panel/blob/c62c4d291745a0b334d8f6e9e3f89b9c4b10366a/docker-compose.yaml#L4 so we could pull in docker-compose-base.yaml and anytime cp-upated.yml updated it we'd know grafana had changed. Right now the job runs monthly, so it would take us a month to figure it out. We could run the job more frequently if needed.